docs/abortRequest.html

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
You also need to update navigation.yml to make sure this new page appears in the doc menu. You can add an entry right after the timeouts.html entry